mercury_engine_data_structures/formats/btunda.py

```diff
@@ -0,0 +1,45 @@
+import construct
+
+from construct import Container, ListContainer
+from mercury_engine_data_structures.formats import BaseResource, standard_format
+from mercury_engine_data_structures.game_check import Game
+
+# btunda has different file versions between launch version and latest release
+VERSION__1_0_0 = 0x02000077
+VERSION__2_1_0 = 0x02000080
+
+class Btunda(BaseResource):
+    """
+    /!\\ /!\\ /!\\ READ THIS WHEN USING!!! /!\\ /!\\ /!\\\n
+    
+    This format has TWO VERSIONS between 1.0.0 and 2.1.0!
+
+    To prevent unexpected behavior across versions, do not use:\n
+    - any key containing EasyMode in Player|CTunablePlayerLifeComponent\n
+    - anything in CTunableBossRushManager
+    """
+    @classmethod
+    def construct_class(cls, target_game: Game, version:int) -> construct.Construct:
+        return standard_format.create('base::tunable::CTunableManager', version)
+
+    @classmethod
+    def parse(cls, data: bytes, target_game: Game) -> "BaseResource":
+        # attempt to parse with 1.0.0 version
```

Comment:
Tbh we could just expand `standard_format.create` to allow the `version` field be any of the known values (aka an enum).
Could make it check if `version` is an int -> const, if a list -> enum that